lib/gds-components/tag/Tag.tsx

```diff
@@ -2,24 +2,18 @@
  * GDS component: https://design-system.service.gov.uk/components/tag/
  */
 
-export type TagLabel = "IN PROGRESS" | "FINISHED" | "NOT STARTED";
+export type TagLabel = "In Progress" | "Finished" | "Not Started";
 
 type TagProps = {
   label: TagLabel;
 };
 
 export const Tag = ({ label }: TagProps): JSX.Element => {
   const colour = {
-    "IN PROGRESS": "blue",
-    FINISHED: "green",
-    "NOT STARTED": "grey",
+    Finished: "green",
+    "In Progress": "light-blue",
+    "Not Started": "grey",
```

Comment:
Looking at the examples, the tags should be sentence case:
![image](https://github.com/digital-land/local-plans-timetable/assets/29425751/a07ec3f3-2cfd-4d20-a41a-f94b8782fbcf)